modules/custom/d_p_gallery/templates/paragraph--d-p-gallery.html.twig

```diff
@@ -55,7 +55,12 @@ view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
       <div class="content-wrapper">
         <div class="container">
           {% block content %}
-            {{ content }}
+            {{ content | without('field_d_cta_link') }}
+            {% if content.field_d_cta_link|render %}
```

Comment:
Ten if to ja bym raczej zrobił `content.field_d_cta_link.0` bo to oznacza tyle, ze jest minimum jeden item w tym polu.
Jak chcesz render to zrób wtedy pełen łańcuszek `|render|striptags|trim` bo nigdy nie wiesz czy output z field_d_cta_link nie zwróci np. pustego diva, zwłaszcza jak ktoś przerobi sobie templatke w swoim theme.